### PR TITLE
New data and time slider

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**
* biome is now temporal
* we have a time slider
* there are three new datasets

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Time Slider](https://trello.com/c/wsPxp9Qk/418-time-slider)

File: .gitignore [hunk 1]
```diff
@@ -24,6 +24,7 @@ speed-measure-plugin*.json
 *.sublime-workspace
 
 # IDE - VSCode
+.vscode
```
Comment (michaelmattig) [inline .gitignore:27]: doesn't that make the next line redundant?

File: projects/wave-core-new/src/lib/project/project.service.ts [hunk 2]
```diff
@@ -598,6 +589,28 @@ export class ProjectService {
         return subject.asObservable();
     }
 
+    /**
+     * Remove all plots from the current project.
+     */
+    clearPlots(): Observable<void> {
+        const subject = new Subject<void>();
```
Comment (michaelmattig) [inline projects/wave-core-new/src/lib/project/project.service.ts:596]: what is this needed for?

**Review by michaelmattig — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (ChristianBeilschmidt) [reply .gitignore]: bb2aaad

File: projects/wave-core-new/src/lib/project/project.service.ts (hunk 2 above)
Comment (ChristianBeilschmidt) [reply projects/wave-core-new/src/lib/project/project.service.ts]: What is it good for? Absolutely nothing.

17498ce

File: .gitignore (hunk 1 above)
Comment (michaelmattig) [reply .gitignore]: why not ignore the folder itself?

Comment (ChristianBeilschmidt) [reply .gitignore]: It's the same, you cannot add an empty folder into git, you would at least create a .keep file.

**Pull request merged**